### ISSUE TITLE
Playerctl not fetching Mozilla Firefox metadata

### ISSUE THREAD

**Issue by rayshabh** [altdesktop/playerctl#293]
Whenever I play video/audio from streaming websites like YouTube, Spotify Web Client, etc. using Mozilla Firefox (v 106.0.1), Playerctl does not fetch any metadata, though the control keys like play/pause actions are working.

I've checked Playerctl with VLC, and it works well and collects the metadata for the same.

Please help to configure Playerctl so that it starts fetching metadata of Firefox too.

Thanks in advance!

**Comment by dylanlobo** [comment]
Not sure Firefox (v 106.0.1) provides any "informative" metadata -- probably does not based on what you've reported above. I've checked with Firefox (v 105) and can confirm that it does not return any useful metadata, so the "issue" is most likely with Firefox. This is all the metadata that it returns when playing a youtube video:
{'mpris:trackid': '/org/mpris/MediaPlayer2/firefox',
 'xesam:album': '',
 'xesam:artist': [''],
 'xesam:title': 'Firefox is playing media'}

**Comment by rayshabh** [comment]
> This is all the metadata that it returns when playing a youtube video: {'mpris:trackid': '/org/mpris/MediaPlayer2/firefox', 'xesam:album': '', 'xesam:artist': [''], 'xesam:title': 'Firefox is playing media'}

Yes, exactly! This is what Firefox returns.

**Comment by code-IM-perfect** [comment]
yeah well it isn't a playerctl issue then...better open it on mozilla bugzilla then
